docs/deprecated.md

```diff
@@ -19,6 +19,12 @@ The following list of features are deprecated in Engine.
 To learn more about Docker Engine's deprecation policy,
 see [Feature Deprecation Policy](https://docs.docker.com/engine/#feature-deprecation-policy).
 
+### Registry v2 schema 1
+
+**Deprecated in Release: v19.03.0**
+
+The registry v2 schema 1 format is deprecated in favor of registry v2 schema 2.
```

Comment:
```
### Pushing to and pulling from registry v2 with schema1 manifest

**Deprecated in Release: v19.03.0**
**Will be removed in Release: v19.09.0**

The registry v2 schema 1 manifest format is deprecated in favor of registry v2 schema 2.
```

Would be good to have a link to schema 1 and schema 2 for completeness.

Reply:
Updated

Reply:
done